frontend/src/scenes/notebooks/Nodes/NotebookNodeMap/NotebookNodeMap.tsx

```diff
@@ -0,0 +1,63 @@
+import { Marker } from 'maplibre-gl'
+
+import { NotebookNodeType } from '~/types'
+import { createPostHogWidgetNode } from 'scenes/notebooks/Nodes/NodeWrapper'
+import { personLogic } from 'scenes/persons/personLogic'
+import { useValues } from 'kea'
+import { LemonSkeleton } from '@posthog/lemon-ui'
+import { NotFound } from 'lib/components/NotFound'
+import { Map } from '../../../../lib/components/Map/Map'
+import { notebookNodeLogic } from '../notebookNodeLogic'
+import { NotebookNodeProps } from 'scenes/notebooks/Notebook/utils'
+import { NotebookNodeEmptyState } from '../components/NotebookNodeEmptyState'
+
+const Component = ({ attributes }: NotebookNodeProps<NotebookNodeMapAttributes>): JSX.Element | null => {
+    const { id } = attributes
+    const { expanded } = useValues(notebookNodeLogic)
+
+    const logic = personLogic({ id })
+    const { person, personLoading } = useValues(logic)
+
+    if (personLoading) {
+        return <LemonSkeleton className="h-6" />
+    } else if (!person) {
+        return <NotFound object="person" />
+    }
+
+    if (!expanded) {
+        return null
+    }
+
+    const longtitude = person?.properties?.['$geoip_longitude']
+    const latitude = person?.properties?.['$geoip_latitude']
+    const personCoordinates: [number, number] | null =
+        !isNaN(longtitude) && !isNaN(latitude) ? [longtitude, latitude] : null
+
+    if (!personCoordinates) {
+        return <NotebookNodeEmptyState message="No map available." />
```

Comment:
This somewhat conflicts with the <NotFound> component that attempts to achieve a similar thing.

I think in this case it makes sense that is shows something different, but not sure it should be a generic component as it is quite specific to this sort of node.